### ISSUE TITLE
small correction

### ISSUE THREAD

**Issue by stas00** [joshfp/pytorch-from-dummy-to-sota#1]
An excellent tutorial, Jose!

A small correction: cell 44, the first conv2d should be:
`nn.Conv2d(1, 16, kernel_size=3, stride=2, padding=1),`

and then if you wish it needs to be updated for the fastai API changes in the recent version
cell 60: ImageItemList => ImageList, and a bunch of other renamed functions which you will see if you re-run the notebook, as "deprecated".

also for the uninitiated a link to how to install fastai would probably be useful, since it's not straightforward. It's best to send the readers here: https://github.com/fastai/fastai/blob/master/README.md#installation

Thank you.

**Comment by joshfp** [comment]
Hello Stas! What a pleasure to receive feedback from you!
You are absolutely right in all your points. I update the notebook accordingly.
Thank you so much!
